### PR TITLE
Fix not overlapping setStreamMetadata prototype

### PR DESCRIPTION
**Pull request by jokesterfr**

### CODE REVIEW

**Comment by jokesterfr** [comment]
Still not working as intended. Will investigate more next week.

**Comment by Vinceveve** [comment]
caution, it's replaced in two different place on eventbuilder

**Review by Vinceveve — COMMENTED** [review]
ExpectedVersion and metadataExpectedVersion are 2 differents values.
one is for the events on the stream.
The other is to avoid writing again metadata.

**Comment by jokesterfr** [comment]
Ok, I reverted this wrong renaming

File: src/cqrs/event-store.publisher.ts [hunk 1]
```diff
@@ -31,13 +31,17 @@ export class EventStorePublisher {
       }
 
       async commit() {
+        if (this.streamMetadata && !this.isMetadataSet) {
+          this.isMetadataSet = true;
```
Comment (Vinceveve) [inline src/cqrs/event-store.publisher.ts:35]: intervertis les lignes

**Review by Vinceveve — APPROVED** [review]

File: src/cqrs/event-store.publisher.ts (hunk 1 above)
Comment (jokesterfr) [reply src/cqrs/event-store.publisher.ts:35]: tu as raison

**Review by Vinceveve — APPROVED** [review]

**Pull request merged**